tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -31,17 +32,11 @@ sub get_script_run {
 
 sub generateXML {
```

Comment:
Please run jobs to make sure prj1/prj3 junit logs are showed correctly and prj1 uploaded guests assets are correct, after this change.

Reply:
Verification run done with PRJ1, which looks good. As long as they keep using the same set of junit log utilities, this will work for all tests straight forward.

Reply:
What is the purpose of this file? I think, if it's a base it should be in `lib/virsh_autotest_base.pm` and this file could be removed.

Reply:
Many test suites are based upon this one. https://openqa.suse.de/tests/overview?distri=sle&version=12-SP5&build=0341&groupid=163

Reply:
OK, I see. But as I said, this seems to be a library - not a test so you should imo get rid of it.

Reply:
Thanks for your advice. But it is beyond the scope of this submission.

Reply:
In my opinion this is exactly the scope of this submission. Here, you introduce `lib/virsh_autotest_base.pm` which is very nice and clean solution. But at the same time, you should get rid of `tests/virt_autotest/virt_autotest_base.pm` which just breaks the design.

Hello @alice-suse, @asmorodskyi, can you please help us here?

Reply:
@pdostal I am afraid not. This may be caused by the confusing base class names. File virt_autotest_base.pm  provides  the base class for our existing QA-SLE virt tests(guest installation, host upgrade, pvusb etc). This base class has many basic facilities which are only used by these tests. File virsh_autotest_base.pm(I actually gave comments already that I do not suggest to use  this name, because it can easily confuse people for whose base class it is. It is better to include this msg in its file name), is the base class for these virt feature tests  that you and leon implemented, such as snapshot,hotplugging, as well as future ones to be implemented, such as storage management, virtual network management etc.
So this file virt_autotest_base.pm should be definitely kept.

Reply:
Now I understand the difference between `ib/virsh_autotest_base.pm` and `tests/virt_autotest/virt_autotest_base.pm` but I still don't undestand why this base **lib**rary is under `tests/` instead of `lib/` directory. In my opinion it should be easy to move it and so make it much more understandable and organized. What do you think about it, @alice-suse?

Reply:
@pdostal - generally agree with you but I think that : 
1. **Maintainer** of the code suppose to have some freedom to decide where and why code is located . I would say there is no 100% clearness in what is "utility code" and where it should be actually located . We have a huge project and different people doing it in different way and I don't feel right to force someone to put something under `lib`
2. In any case I would say this is not related to this PR because `virt_autotest_base.pm` was not introduced by this PR so this could be just a nice idea for some other PR